src/js/calculator.js.js

```diff
@@ -0,0 +1,88 @@
+import {$total} from "./index.js";
+import {
+  PLUS,
+  MINUS,
+  MULTIPLICATION,
+  DIVISION,
+  ALERT_MESSAGE,
+} from "./constants/constants.js";
+
+const operators = {
+  [PLUS]: (a, b) => a + b,
+  [MINUS]: (a, b) => a - b,
+  [MULTIPLICATION]: (a, b) => a * b,
+  [DIVISION]: (a, b) => Math.floor(a / b),
+};
+const keyOfOperators = Object.keys(operators);
+
+const lengthChecker = (() => {
+  let length = 0;
+
+  return {
+    increase() {
+      return length++;
+    },
+
+    check() {
+      return length;
+    },
+
+    reset() {
+      return (length = 0);
+    },
+  };
+})();
+
+const putResult = () => {
+  const method = $total.textContent
+    .split("")
+    .find((v) => keyOfOperators.includes(v));
+
+  const firstNumber = $total.textContent.split(method).shift();
+  const secondNumber = $total.textContent.split(method).pop();
```

Comment:
```suggestion
  const [firstNumber, secondNumber] = $total.textContent.split(method).map(Number);
```
이 부분은 이렇게 표현해도 좋을 것 같아요!